.circleci/config.yml

```diff
@@ -131,7 +131,7 @@ commands:
             # Set the installed version as the default one
             nvm alias default $(nvm current)
             # Enforce to use the default version in the rest of workflow
-            echo 'nvm use default' >> $BASH_ENV
+            echo 'nvm use default --silent' >> $BASH_ENV
```

Comment:
I decided to silent the output of this command because it was logging `Now using node v14.17.6 (npm v6.14.15)` on every `run` step of the workflows. Besides, for the "Check correctness" workflow, it was logging it two times ([reference](https://app.circleci.com/pipelines/github/wordpress-mobile/gutenberg-mobile/14158/workflows/309e8e7d-b45f-4a46-985b-ed5bd060f1e2/jobs/77114?invite=true#step-110-3)), one when running the "Run checks" job step and then when running the Bash script `bin/ci-checks-js.sh`.

Reply:
No strong feelings but I wonder if the versions callout could be helpful during debugging of the jobs (when needed) and the output is not too verbose.

Yes, the double output in `bin/ci-checks-js.sh` is not great, but not too bad anyway. If anything, I'd try to figure out how to optimize to only have to call `nvm` once in that script, but feels like out of scope for this PR.

Reply:
> No strong feelings but I wonder if the versions callout could be helpful during debugging of the jobs (when needed) and the output is not too verbose.

Right, it could be useful for debugging, I'll revert the change.

Reply:
> Yes, the double output in bin/ci-checks-js.sh is not great, but not too bad anyway. If anything, I'd try to figure out how to optimize to only have to call nvm once in that script, but feels like out of scope for this PR.

I tried to debug this by connecting via SSH to CircleCI but I couldn't figure out a way to prevent it, it would be great if we investigate it further in the future although I think it's a minor issue.

Reply:
This suggestion was added in [this commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3990/commits/d0fdfa122f3c568241360dcb9c890a30d43fa18d).